architecture/overview.adoc

```diff
@@ -5,22 +5,68 @@
 :icons:
 :experimental:
 
-OpenShift is a layered system designed to expose underlying Docker and Kubernetes concepts as accurately as possible, with a focus on easy composition of applications by a developer (install Ruby, push code, add MySQL). Unlike 2.x, more flexibility of configuration is exposed after creation in all aspects of the model. The concept of an application as a separate object is being removed in favor of more flexible composition of "services" - allowing two web containers to reuse a DB, or expose a DB directly to the edge of the network.
+OpenShift is a layered system designed to expose underlying
+https://www.docker.com/[Docker] and
+http://kubernetes.io/[Kubernetes] concepts as accurately as possible, with a focus on easy composition
+of applications by a developer (install Ruby, push code, add MySQL). Unlike
+OpenShift 2.x, more flexibility of configuration is exposed after creation
+in all aspects of the model. The concept of an
+link:../whats_new/applications.html[application] as a separate object
+is being removed in favor of more flexible composition of "services", allowing
+two web containers to reuse a database or expose a database directly to the
+edge of the network.
 
 ==== What are those layers?
```

Comment:
- Thoughts on heading level? `==` maybe? Since it's an Overview topic it doesn't have a TOC, so ultimately it's just a matter of how it looks rendered.

- How about "What are the layers?" instead of "those"?

- Should this be title cased, or no since it's phrased as a sentence/question?